content/tutorials/mapping.md

```diff
@@ -90,13 +98,40 @@ $ rosrun map_server map_saver -f map_name map:=/car/map
 ```
 This should create a .pgm file with the map and a .yaml file with the map metadata. 
 
-To load your new map into the simulator, move both files to `~/catkin_ws/src/mushr_sim/maps/` and edit `~/catkin_ws/src/mushr_sim/launch/map_server.launch` to set the map to the name of the .yaml file.
+### Using the map
```

Comment:
could you add a section for "In Sim" and "On Robot" because there are two map server files?